src/main/scala/org/broadinstitute/dsde/firecloud/webservice/SnapshotApiService.scala

```diff
@@ -19,7 +19,10 @@ trait SnapshotApiService extends FireCloudDirectives with UserInfoDirectives {
       pathPrefix("workspaces" / Segment / Segment / "snapshots") { (namespace, name) =>
         pathEnd {
           post {
-            passthrough(SnapshotApiService.createDataRepoSnapshotURL(namespace, name), HttpMethods.POST)
+            passthrough(SnapshotApiService.dataRepoSnapshotBaseURL(namespace, name), HttpMethods.POST)
+          } ~
+          get {
+            passthrough(SnapshotApiService.dataRepoSnapshotBaseURL(namespace, name), HttpMethods.GET)
```

Comment:
I don't think the `limit` and `offset` query params are getting passed through to rawls here

Reply:
whoops, good catch!